docs/content/latest/yugabyte-cloud/cloud-network/cloud-vpc-setup.md

```diff
@@ -0,0 +1,114 @@
+
+VPCs are configured on the [VPCs](../cloud-add-vpc/) page of the **VPC Network** tab on the **Network Access** page.
+
+Peering connections are configured on the [Peering Connections](../cloud-add-peering) page of the **VPC Network** tab on the **Network Access** page.
+
+## Before you begin
+
+Before setting up the VPC network, you will need the following:
+
+- The details of the VPN you want to peer with, including
+
+  - GCP - the project ID and the network name, and CIDR block.
+  - AWS - the AWS account ID, and the VPC ID, region, and CIDR block.
+
+## Create a VPC in Yugabyte Cloud
+
+1. On the **Network Access** page, select **VPC Network**, then **VPCs**.
+1. Cick **Create VPC** to display the **Create VPC** sheet.
+1. Enter a name for the VPC.
+1. Choose the provider (AWS or GCP).
+1. If you selected AWS, select the region. VPCs for GCP are created globally and assigned to all regions supported by Yugabyte Cloud.
+1. Specify the VPC CIDR address. You can use the range suggested by Yugabyte Cloud, or enter a custom IP range. The IP range cannot overlap with another VPC in your cloud.
+1. Click **Save**.
+
+## Deploy a cluster in the VPC
+
+Once a VPC has been created, you can [deploy your cluster](../../cloud-basics/create-clusters/) in the VPN.
+
+1. On the **Clusters** page, click **Add Cluster**.
+1. Choose **Yugabyte Cloud** and click **Next**.
+1. Choose the provider you used for your VPC.
+1. Enter a name for the cluster.
+1. Select the **Region**. For AWS, choose the same region as your VPC.
+1. Set the Fault Tolerance. For production clusters, typically this will be Availability Zone Level.
+1. Under **Network Access**, choose **Deploy this cluster in a dedicated VPC**, and select your VPC.
+1. Click **Create Cluster**.
+
+## Create a Peering Connection in Yugabyte Cloud
+
+Once you have created at least one VPC in Yugabyte Cloud, you can create a peering connection with an application VPC on the same cloud provider. You need to specify the following:
+
+- The Yugabyte Cloud VPC to peer.
+- Details of the VPC you want to peer with, including:
+  - GCP - the project ID and the network name.
+  - AWS - the AWS account ID, and the VPC ID, region, and CIDR block.
+
+To create a peering connection, do the following:
+
+1. On the **Network Access** page, select **VPC Network**, then **Peering Connections**.
+1. Click **Create Peering** to display the **Create Peering** sheet.
+1. Enter a name for the peering connection.
+1. Choose the provider you used for your VPC.
+1. Choose the Yugabyte Cloud VPC. Only VPCs that use the same provider are listed.
+1. Enter the application VPC information for the provider you selected.
+1. Select **Add application CIDR to IP allow list** to add the the CIDR range to your cloud IP allow list. You will add this IP allow list to your cluster so that the application VPC can connect to the database.
```

Comment:
Where is this option in the produt?

Reply:
This was in the mockups; I wasn't sure if it was going to be added or not, will remove if not